.eslintrc

```diff
@@ -16,8 +16,12 @@ rules:
     - avoid-escape
   semi:
     - error
-    - never
+    - always
```

Comment:
Precisely. Otherwise I'd have to refactor in a way that touches nearly every line the codebase, jamming up `git blame`.